### ISSUE TITLE
is the implementation parallel?

### ISSUE THREAD

**Issue by treasureqqm** [pdollar/coco#532]
These days I found my coco evaluation time for object detection task differs a lot, ranging from 4 minutes to 40 minutes or so. And I also found when coco is doing evaluation, there are many threads appeared (see below picture, the screeshot of "top" command). So i wonder if the coco evaluation is implemented parallel. If I have any misunderstanding, please let me know. Thanks all.
![image](https://user-images.githubusercontent.com/33859606/126462395-1796c168-3234-486a-b1f2-7492dd14357b.png)

**Comment by developer0hye** [comment]
@treasureqqm Did you solve the problem?

**Comment by treasureqqm** [comment]
> @treasureqqm Did you solve the problem?

No, it's the problem of my machine. If i use another machine, all computation is normal. But i also think that the underlying library which cocoapi is built on (such as numpy) is implemented parallel.

**Comment by developer0hye** [comment]
@treasureqqm Thanks for your reply!
